tests/integration/features/one-to-one.feature

```diff
@@ -9,6 +9,11 @@ Feature: one-to-one
     Then user "participant2" is participant of the following rooms
     Then user "participant3" is participant of the following rooms
 
+  Scenario: User1 invites user1 ... that should not work ...
```

Comment:
I would remove the _... that should not work ..._ part from the scenario name, as it is not consistent with the rest of scenario names; to document that it should not work I would just add a comment with _#_.